### ISSUE TITLE
Figure out how to set up automatic testing 😅 

### ISSUE THREAD

**Issue by carodew** [GSA/10x#28]
## it should include …
- [ ] ping someone in #dev and see if there's someone who can coach us through this
- [ ] figure out what Federalist is testing for already (so we don't duplicate) 
- [ ] talking to someone who knows what they're doing (Circle CI, accesslint?) 
- [ ] creating clear issues for the work that needs to be done, once we know what that work is

## it's done when …
- [ ] we know what we need in terms of automatic testing, and that info has been added to #31 

**Does not include** actually setting up the automatic testing, we'll do that in a separate issue. 

## outstanding questions
- [ ] how do we even ...

**Comment by carodew** [comment]
## what I've learned so far 

plainlanguage.gov uses a Circle CI implementation that’s a very basic html check plus a few other things and would probably work fine for us. 

This is the html proofer – https://github.com/gjtorikian/html-proofer
and this is plainlanguage.gov's circle ci implementation –https://github.com/GSA/plainlanguage.gov/blob/master/circle.yml 

Unless someone else object, I think this is probably our best path forward and is probably `good enough` – I can dig into _how_ we go about implementing it if I hear no objections. You know what they say, if it's good enough for plainlanguage.gov, it's good enough for me.

**Comment by jeremyzilar** [comment]
OMG, thank you for digging this up. 
Maybe I can ask @hursey013 how he set this up.

**Comment by carodew** [comment]
@jeremyzilar All I did was ask in #dev. :D 

and @hursey013 is the person who is giving me tips. :D 

He sez – "if you have a config.yml in the root of your project, then you basically just connect your github account with circleci, add the project repo, hit build, and it’ll find the circle.yml"

**Comment by carodew** [comment]
So I've merged the `circle.yml` file from plainlanguage.gov into demo (probably needs to be merged to prod to but I think whenever the next release is will be fine) and I'm able to run builds in the Circle CI interface. 

I do get an error bc [htmlproofer](https://github.com/gjtorikian/html-proofer) is not installed, but it's called in our circle.yml file. I think once someone does that (@thisisdano maybe?) we will be good! 

I will update the "install automatic tests" ticket with these details and move the card into Ready. 

(I'm only doing this on a Saturday because I realized I left this half done yesterday and figured it'd only take five minutes).
